src/ReadCSVFile.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Created by ken-linux2 on 14/07/16.
+ */
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+
+public class ReadCSVFile {
+    public static void main(String[] args) {
+        ReadCSVFile RCF = new ReadCSVFile();
+        RCF.run();
+    }
+
+    private void run() {
+        String csvFile = "/home/ken-linux2/Downloads/macycExampleFile.csv - macycExampleFile.csv.csv";
```

Comment:
put "final" before parameters that are not going to change in the method. It will verify that no one will change the parameter.